other/perun-propagate/etc/perun_propagate

```diff
@@ -29,6 +32,16 @@ fi
 # Load lsb functions
 . /lib/lsb/init-functions
 
+# Return 0 if process with PID is still runing, 1 if not
+is_process_running() {
+	kill -0 "$1" 2>/dev/null 1>&2
+	if [ "$?" -eq 0 ]; then
+		return 0;
+	else
+		return 1;
+	fi
```

Comment:
Could be written in more compact way.
return $? # may return other values then 1 for false, but it usually doesn't matter.

or

kill  -0 "$1" 2>/dev/null 1>&2 && return 0 || return 1